packages/toolpad-utils/src/fs.ts

```diff
@@ -3,8 +3,21 @@ import * as path from 'path';
 import { Dirent } from 'fs';
 import * as yaml from 'yaml';
 import { yamlOverwrite } from 'yaml-diff-patch';
+import * as prettier from 'prettier';
 import { errorFrom } from './errors';
 
+/**
+ * Formats a yaml source with `prettier`.
+ */
+
+export async function formatYaml(code: string, filePath: string): Promise<string> {
```

Comment:
No need to export it for now

```suggestion
async function formatYaml(code: string, filePath: string): Promise<string> {
```